common/data_format.hpp

```diff
@@ -35,18 +35,34 @@ namespace ILLIXR {
 		ullong dataset_time;
 	} imu_cam_type;
 
+	typedef struct {
+		Eigen::Matrix<double,3,1> w_hat;
+		Eigen::Matrix<double,3,1> a_hat;
+		Eigen::Matrix<double,3,1> w_hat2;
+		Eigen::Matrix<double,3,1> a_hat2;
+		Eigen::Matrix<double,13,1> pose;
```

Comment:
Isn't this is `state` (based on [this line](https://github.com/ILLIXR/open_vins/compare/50d1d5a2869b8d6bc9a276e984d1a2498c59c077...e27874535235c44d9999dfd383cc8bc7170b0397?diff=split#diff-18e35e680b5b2b4270f745b227d7aa49R189))? That is, why call it `pose`?